src/ir/comp.rs

```diff
@@ -1353,9 +1385,14 @@ impl CompInfo {
                         cur.kind() != CXCursor_EnumDecl
                     {
                         let ty = cur.cur_type();
+                        let public = cur.access_specifier() ==
+                            clang_sys::CX_CXXPublic ||
```

Comment:
please call access_specifier() just once. It's an ffi call after all so it's not like the compiler can remove redundant calls to it.